### PR TITLE
Benchmarks ogs

### PR DESCRIPTION
**Pull request by jbathmann**
This PR is adding a new module benchmark for the belowground-competition-concept "OGSLargeScale3D".
Boundary conditions are set according to the description on homepage. Initial population equals the initial population of other benchmarks.
Soil permeability is set to 2.5e-12 as a strong effect of tree water uptake is observed in this case (in combination with a hydraulic gradient of -0.1/22).
Adaptive timestepping is chosen as the initial pyMANGA timesteps require more iterations in the ogs simulation.

### CODE REVIEW

**Review by mcwimm — COMMENTED** [review]
Thanks (also for fixing my messed up paths)!
Tested it (with 3 time steps only as it takes quite long).

In general I'm a bit confused by the python_script: in my IDE it shows me that most variables, packages are not used. Some comments would help here :)

File: Benchmarks/ModuleBenchmarks/BelowgroundCompetition/OGSLargeScale3D/python_script.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    return float(m * x)
+
+def pressure_value(z,x):
+    return 1000 * 9.81 * (transectElevation(x) - z)
+
+## Dirichlet BCs for pressure
+class BC_p_D(OpenGeoSys.BoundaryCondition):
+
+    def getDirichletBCValue(self, t, coords, node_id, primary_vars):
+        x, y, z = coords
```
[the remark this answers is not in the document]
Comment (jbathmann) [reply Benchmarks/ModuleBenchmarks/BelowgroundCompetition/OGSLargeScale3D/python_script.py:15]: removed

File: Benchmarks/ModuleBenchmarks/BelowgroundCompetition/OGSLargeScale3D/python_script.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import OpenGeoSys
+import vtk as vtk
+import numpy as np
```
[the remark this answers is not in the document]
Comment (jbathmann) [reply Benchmarks/ModuleBenchmarks/BelowgroundCompetition/OGSLargeScale3D/python_script.py:3]: reduced

File: Benchmarks/ModuleBenchmarks/BelowgroundCompetition/OGSLargeScale3D/python_script.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        x, y, z = coords
+        value = pressure_value(z, x)
+        return (True, value)
+
+## Dirichlet BCs for concentration
+class BC_left_C_D(OpenGeoSys.BoundaryCondition):
+
+    def getDirichletBCValue(self, t, coords, node_id, primary_vars):
+        x, y, z = coords
+        value = left_salinity
```
[the remark this answers is not in the document]
Comment (jbathmann) [reply Benchmarks/ModuleBenchmarks/BelowgroundCompetition/OGSLargeScale3D/python_script.py:27]: comment added

File: Benchmarks/ModuleBenchmarks/BelowgroundCompetition/OGSLargeScale3D/python_script.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+def pressure_value(z,x):
+    return 1000 * 9.81 * (transectElevation(x) - z)
+
+## Dirichlet BCs for pressure
+class BC_p_D(OpenGeoSys.BoundaryCondition):
+
+    def getDirichletBCValue(self, t, coords, node_id, primary_vars):
+        x, y, z = coords
+        value = pressure_value(z, x)
+        return (True, value)
```
[the remark this answers is not in the document]
Comment (jbathmann) [reply Benchmarks/ModuleBenchmarks/BelowgroundCompetition/OGSLargeScale3D/python_script.py:19]: comment added

**Pull request merged**